data-pipeline-flink/dp-core/src/main/scala/org/sunbird/dp/core/BaseJobConfig.scala

```diff
@@ -30,6 +30,7 @@ class BaseJobConfig(val config: Config, val jobName: String) extends Serializabl
     val properties = new Properties()
     properties.setProperty("bootstrap.servers", kafkaBrokerServers)
     properties.setProperty("group.id", groupId)
+    //properties.setProperty("auto.offset.reset", "earliest") // Uncomment this line to run the test flink job
```

Comment:
Does this have any issue with the main flink jobs run?

Reply:
Main flink job does not have any issue and it by default it will take from the "latest" offset but for embedded kafka we must pass "earliest" offset as value else it was not reading any message from the output topic.